core/src/data_sources/data_source.rs

```diff
@@ -505,6 +526,12 @@ impl DataSource {
                 &tags_path,
                 "application/json",
             ),
+            Object::create(
```

Comment:
Let's not store it in GCS. I will take stopping storing the tags and timestamp in another PR.

The idea was to be able to reconstruct everything from GCS but I think it creates more complexity than safety (we can reconstruct from the DB + only the content in GCS) and the event that we loose the DB but not GCS is very unlikely as we have daily backups of it.